.github/workflows/update.yml

```diff
@@ -31,70 +31,153 @@ jobs:
+      id: redhat-2003-2008
       run: ./vuln-list-update -target redhat -years 2003,2004,2005,2006,2007,2008
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2009-2011
+      id: redhat-2009-2011
       run: ./vuln-list-update -target redhat -years 2009,2010,2011
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2012
+      id: redhat-2012
       run: ./vuln-list-update -target redhat -years 2012
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2013
+      id: redhat-2013
       run: ./vuln-list-update -target redhat -years 2013
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2014
+      id: redhat-2014
       run: ./vuln-list-update -target redhat -years 2014
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2015
+      id: redhat-2015
       run: ./vuln-list-update -target redhat -years 2015
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2016
       run: ./vuln-list-update -target redhat -years 2016
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2017
+      id: redhat-2017
       run: ./vuln-list-update -target redhat -years 2017
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2018
+      id: redhat-2018
       run: ./vuln-list-update -target redhat -years 2018
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2019-2020
+      id: redhat-2019-2020
       run: ./vuln-list-update -target redhat -years 2019,2020
+      continue-on-error: true
+
+    - name: Check for failures and Send a message to Microsoft Teams
+      uses: luisghz/simple-ms-teams-webhook-notifier@v1-latest
+      if: >
+        steps.nvd.outputs.status != 'success' ||
+        steps.alpine.outputs.status != 'success' ||
+        steps.debian.outputs.status != 'success' ||
+        steps.debian-oval.outputs.status != 'success' ||
+        steps.ubuntu.outputs.status != 'success' ||
+        steps.amazon.outputs.status != 'success' ||
+        steps.oracle-oval.outputs.status != 'success' ||
+        steps.redhat-oval.outputs.status != 'success' ||
+        steps.photon.outputs.status != 'success' ||
+        steps.ghsa.outputs.status != 'success' ||
+        steps.cwe.outputs.status != 'success' ||
+        steps.suse-cvrf.outputs.status != 'success' ||
+        steps.redhat-1996-2002.outputs.status != 'success' ||
+        steps.redhat-2003-2008.outputs.status != 'success' ||
+        steps.redhat-2012.outputs.status != 'success' ||
+        steps.redhat-2013.outputs.status != 'success' ||
+        steps.redhat-2014.outputs.status != 'success' ||
+        steps.redhat-2015.outputs.status != 'success' ||
+        steps.redhat-2016.outputs.status != 'success' ||
+        steps.redhat-2017.outputs.status != 'success' ||
+        steps.redhat-2018.outputs.status != 'success' ||
+        steps.redhat-2019-2020.outputs.status != 'success'
```

Comment:
I couldn't find a way to be able to classify and gather the error on a job level but only on a step level. Hence we need these conditions. Any suggestions are welcome if this is unacceptable. 

To me since this logic will rarely change, I think it's fine as is.

Reply:
Can we know which job fails?

Reply:
After reading the docs, I couldn't find a way.